GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -206,7 +206,7 @@ private extension EntryWidget {
                 case .secureMessaging:
                     try self.environment.engagementLauncher.startSecureMessaging()
                 case .callVisualizer:
-                    break
+                    Glia.sharedInstance.callVisualizer.resume()
```

Comment:
Why singleton has to be used here?

Reply:
Callback added [478149b](https://github.com/salemove/ios-sdk-widgets/pull/1154/commits/478149b78108faab8c81a1748697e8301b278675)